docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -103,6 +103,14 @@ children:
       url: Parsing-requests.html
       output: 'web, pdf'
 
+  - title: 'Middleware'
+    url: Middleware.html
+    output: 'web, pdf'
+
+  - title: 'Using Express Middleware'
+    url: Express-middleware.html
+    output: 'web, pdf'
```

Comment:
I think this page should be nested inside [Sequence](https://loopback.io/doc/en/lb4/Sequence.html). Please note that the Sequence page already provides some content related to Express-middleware, see [Working with Express middleware](https://loopback.io/doc/en/lb4/Sequence.html#working-with-express-middleware).

Please update that existing content to work well with your new pages and to avoid duplicative and/or contradicting information.

_I have also briefly considered the option to put this page into [Usage scenarios](https://loopback.io/doc/en/lb4/Usage-scenarios.html), but since we already have some middleware-related content in Sequence, it may be best to put the new content there too._

Reply:
Also, do you have any plans for https://loopback.io/doc/en/lb4/express-with-lb4-rest-tutorial.html? Should we update it to mention the newer (and better) way for mounting Express middleware? Perhaps we should deprecate and eventually remove that page?

Reply:
> I think this page should be nested inside Sequence. Please note that the Sequence page already provides some content related to Express-middleware, see Working with Express middleware.

We might have to use this page for two cases:

- Apply Express middleware in the sequence
- Apply Express middleware as interceptors